skyvern/forge/sdk/db/client.py

```
@@ -1816,6 +1828,413 @@ async def soft_delete_workflow_by_permanent_id(
+        self,
+        workflow_permanent_id: str,
+        organization_id: str,
+        folder_id: str | None,
+    ) -> Workflow | None:
+        """Update folder assignment for the latest version of a workflow."""
+        try:
+            # Validate folder exists if folder_id is provided
+            if folder_id:
+                folder = await self.get_folder(folder_id, organization_id)
+                if not folder:
+                    raise ValueError(f"Folder {folder_id} not found")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)